modules/slo-generator/main.tf

```diff
@@ -137,3 +136,19 @@ resource "google_cloud_run_service" "service" {
     google_secret_manager_secret_version.secret-version-data
   ]
 }
+
+resource "google_artifact_registry_repository" "slo-generator-repo" {
```

Comment:
Do we need to add infra to mirror ghcr or could we just update default to point to ghcr.io image?

Reply:
Hey @bharathkkb - This is actually why this PR go "stuck" for so long.

Cloud Run only supports AR and Dockerhub directly, and now recommends to use an AR remote for anything else:
https://cloud.google.com/run/docs/deploying#images

Reply:
Ah I wasn't aware in that case we can keep it in.